torch/sparse/semi_structured/__init__.py

```diff
@@ -0,0 +1,79 @@
+import torch
+from .semi_structured_sparse_tensor import SemiStructuredSparseTensor
+import random
+
+__all__ = [
+    "to_semi_structured_sparse",
+    "is_semi_structured_sparse",
+    "gen_semi_structured_sparse_mask",
+    "SemiStructuredSparseTensor",
+]
+
+def gen_semi_structured_sparse_mask(r, c, dtype=torch.float16, device="cuda"):
```

Comment:
Do we really need this to be part of our API or is this more for testing?

The docs and API name also don't make it clear that this is a random mask.